docusaurus.config.js

```diff
@@ -6,20 +6,20 @@ const lightCodeTheme = require('prism-react-renderer/themes/github')
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'WalletConnect Docs',
+  title: 'WalletConnect Specs',
   tagline: 'WalletConnect is an open protocol to communicate securely between Dapps and Wallets.',
-  url: 'https://docs.walletconnect.com/',
+  url: 'https://specs.walletconnect.com/',
   baseUrl: '/',
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'throw',
   favicon: 'img/favicon.ico',
   organizationName: 'walletconnect',
-  projectName: 'walletconnect-docs',
+  projectName: 'walletconnect-specs',
   scripts: [
     {
       src: 'https://plausible.io/js/plausible.js',
       defer: true,
-      'data-domain': 'docs.walletconnect.com'
+      'data-domain': 'specs.walletconnect.com'
```

Comment:
Not sure if this domain needs to be created in Plausible or not. Do you know @chadyj ?